### ISSUE TITLE
Part of alerts from Alertmanager not displayed if severity not describe all possible options in Nagstamon datasource

### ISSUE THREAD

**Issue by dragoangel** [HenriWahl/Nagstamon#900]
Nagstamon 3.10.1 on Windows (and may be on other OS) have a bug:
if in Alertmanager you have severity that you not configured in Alertmanager server properties in Nagstamon then not only this alert will not be displayed, but also alerts "after" this alert will be missing.

F.e.: 
- alertmanager have 6 firing alerts, 2 with severity warning, 2 with critical and one with none and one with info
- in this case with default configuration of alertmanager in Nagstamon you should see 4 alerts (2 warning and 2 critical)

But due to bug you could not see part this 4 alerts. If you configure alertmanager in Nagstamon to understand none and info severity you will all 6 alerts as should.

**Comment by HenriWahl** [comment]
@dragoangel sorry for the late reply: can you please test the latest testing release?

**Comment by dragoangel** [comment]
Hi @HenriWahl, is there any build installers for test release or docs how to build them under windows by myself? Thank you in advance

**Comment by HenriWahl** [comment]
@dragoangel yes at https://github.com/HenriWahl/Nagstamon/releases/tag/latest you will always find the latest testing releases for all platforms.

**Comment by dragoangel** [comment]
> @dragoangel yes at https://github.com/HenriWahl/Nagstamon/releases/tag/latest you will always find the latest testing releases for all platforms.

Installed latest version, removed `none` from `Map to UNKNOWN` and my `severity="critical"` was hidden. Adding back `none` reveal my `critical` alert back. `severity=none` is default severity in `kube-prometheus-stack` for `Watchdog` and `InfoIngibitor` alerts. Also I not 100% sure, but `Map to OK` isn't needed on alertmanager as API not contains resolved alerts, right?

**Comment by dragoangel** [comment]
resolved in 3.18.0

**Comment by HenriWahl** [comment]
@dragoangel thanks for reporting
